packages/ui/origin-ui-core/src/components/organization/organizationMenu.ts

```diff
@@ -78,8 +79,9 @@ export const useOrganizationMenu = (): IOrganizationMenuItem[] => {
             key: 'register-irec',
             label: 'Register I-REC',
             component: IRECRegisterForm,
-            feature: OriginFeature.IRec,
             show:
+                config &&
```

Comment:
Sometimes context returned null. This may be due to some problems with rerendering and such